src/protobuf/bmv2/program.proto

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "src/protobuf/bmv2/header.proto";
+import "src/protobuf/bmv2/parser.proto";
+import "src/protobuf/bmv2/table.proto";
+
+
+// Specifies the overal structure of a p4 program json
```

Comment:
Lets follow the style guide and use full sentences that end with a full stop "." (see https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar). Might seem like a nitpick, but it's good to get into the habit of writing clear and easy to read comments :)